chapters/0-introduction.tex

```diff
@@ -0,0 +1,354 @@
+Original data, especially those collected or assembled by the team themsselves,
+requires that the team carefully document how the data was created, handled, and analyzed.
+These tasks now contribute as much weight to the quality of the evidence
+as the research design and the statistical approaches do.
+At the same time, the scope and scale of empirical research projects is expanding:
+more people are working on the same data over longer timeframes.
+For that reason, the central premise of this book is that data work is a ``social process''.
+This means that the many different people on a team need to have the same ideas
+about what is to be done, and when and where and by whom,
+so that they can collaborate effectively on a large, long-term research project.
+
+In the past, these processes were often treated as a ``black box'' in research.
+A published manuscript might exhaustively detail
+research designs, estimation strategies, and theoretical frameworks,
+but would typically reserve very little space for detailed descriptions
+of how data was actually collected and handled.
+Not only is it almost impossible to assess the quality of the data in such a paper,
+it is very hard for research teams -- particularly new staff --
+to understand and learn the skills and tools needed to do this well.
+There are few guides to these conventions, standards, and best practices
+that are fast becoming a necessity for empirical research.
+Since 2010, reproducibility practices in development have been rapidly adopted,\cite{swanson2020research}
+in part due to increasing requirements by publishers and funders to release code and data.
+However, little practical guidance on complete data handling workflows are available for practitioners,
+aside from one relatively recent handbook on reproducibility.\cite{christensen2019transparent}
+This book aims to fill that gap.
+It covers data workflows at all stages of the research process,
+from design to data acquisition and analysis.
+
+The Analytics team has invested many hours over many years
+learning from data work across DIME's portfolio,
+identifying inefficiencies and barriers to success,
+developing tools and trainings, and standardizing best-practice workflows at DIME.
+It has also invested significant energy in the language and materials
+used to teach these workflows to new team members,
+and, in many cases, in software tools that support these workflows explicitly.
+DIME team members often work on diverse portfolios of projects
+with a wide range of teammates, and we have found
+that standardizing core processes across all projects
+results in higher-quality work with fewer opportunities to make mistakes.
+In that way, the Analytics team is DIME's method of ``institutionalizing''
+tools and practices, developed and refined over time,
+that give the department a common base of knowledge and practice.
+In 2018 and 2019, for example, DIME adoped universal reproducibility checks
```

Comment:
```suggestion
In 2018, for example, DIME adoped universal reproducibility checks
```
Even if it is more correct to say _2018 and 2019_ for some reason, can't we just say 2018 to make it flow better?